javascript/src/game.js

```diff
@@ -1,171 +1,173 @@
-const Game = function () {
-  var players          = new Array();
-  var places           = new Array(6);
-  var purses           = new Array(6);
-  var inPenaltyBox     = new Array(6);
+const Game = function() {
+  this.logs = [];
+  this.log = text => {
+    console.log(text);
+    this.logs.push(text);
+  };
+
+  var players = new Array();
+  var places = new Array(6);
+  var purses = new Array(6);
+  var inPenaltyBox = new Array(6);
 
-  var popQuestions     = new Array();
+  var popQuestions = new Array();
   var scienceQuestions = new Array();
-  var sportsQuestions  = new Array();
-  var rockQuestions    = new Array();
+  var sportsQuestions = new Array();
+  var rockQuestions = new Array();
 
-  var currentPlayer    = 0;
+  var currentPlayer = 0;
   var isGettingOutOfPenaltyBox = false;
 
-  var didPlayerWin = function(){
-    return !(purses[currentPlayer] == 6)
+  var didPlayerWin = () => {
+    return !(purses[currentPlayer] === 6);
   };
 
-  var currentCategory = function(){
-    if(places[currentPlayer] == 0)
-      return 'Pop';
-    if(places[currentPlayer] == 4)
-      return 'Pop';
-    if(places[currentPlayer] == 8)
-      return 'Pop';
-    if(places[currentPlayer] == 1)
-      return 'Science';
-    if(places[currentPlayer] == 5)
-      return 'Science';
-    if(places[currentPlayer] == 9)
-      return 'Science';
-    if(places[currentPlayer] == 2)
-      return 'Sports';
-    if(places[currentPlayer] == 6)
-      return 'Sports';
-    if(places[currentPlayer] == 10)
-      return 'Sports';
-    return 'Rock';
+  const categoryLookup = {
```

Comment:
Looks very array like, just didn't get to updating this refactor from an object to an array.